app/controllers/resources_controller.rb

```diff
@@ -160,10 +160,7 @@ def metadata_for_file(globus_file, file)
     raise GlobusNotFoundError, "Globus file [#{globus_file}] not found." unless File.exist?(globus_file)
 
     file[:size] = File.size(globus_file)
-    file[:hasMessageDigests] = [
-      { type: 'md5', digest: Digest::MD5.file(globus_file).hexdigest },
-      { type: 'sha1', digest: Digest::SHA1.file(globus_file).hexdigest }
-    ]
+    file[:hasMessageDigests] = [] # calculated during UpdateJob
```

Comment:
And this needs to be in the controller so that `file[:hasMessageDigests]` is empty, which https://github.com/sul-dlss/sdr-api/pull/556/files#diff-a6913203d143db3d0665441b9a70307c6ee42012b9bc68d5508f2fa3068b338cR57 checks, right? If so, I wonder if tossing that into the comment might help. I can imagine someone (*cough* probably me) looking at this code in the future and thinking "oh, I bet I can delete this..."

Reply:
@mjgiarlo thanks for noticing this, I could totally see wanting to remove that empty list, and it would break the digest generation. I decided to use `blank?` instead to check if `file[:hasMessageDigests]` has anything in it. This means that supplying an empty list is no longer required.

Reply:
Perfect 😃